src/tallies/TallyBase.C

```diff
@@ -44,6 +44,13 @@ TallyBase::validParams()
       "Auxiliary variable name(s) to use for OpenMC tallies. "
       "If not specified, defaults to the names of the scores");
 
+  params.addParam<std::vector<SubdomainName>>(
+      "block",
+      "Subdomains for which to add tallies in OpenMC. If not provided, "
+      "tallies will be applied over the entire mesh.");
```

Comment:
```suggestion
      "tallies will be applied over the entire domain corresponding to the [Mesh] block.");
```

Reply:
I feel like this is slightly more specific and might be helpful for people using cell tallies (I could see "entire mesh" leading to some confusion with what role the mesh plays for cell tallies). Feel free to disregard, since the phrasing was already there before